python/lsst/ts/MTMount/mtmount_csc.py

```diff
@@ -402,28 +521,17 @@ async def send_commands(self, *commands, dolock=True):
     async def camera_cable_wrap_loop(self):
         self.log.info("Camera cable wrap control begins")
         try:
-            self.prev_rot_application = None
+            self.rotator_position_updated.clear()
             while True:
-                rot_application = await self.rotator.tel_Application.next(flush=True)
-                estimated_velocity = 0
-                if self.prev_rot_application is not None:
-                    dt = (
-                        rot_application.private_sndStamp
-                        - self.prev_rot_application.private_sndStamp
-                    )
-                    if dt < MAX_ROTATOR_APPLICATION_GAP:
-                        estimated_velocity = (
-                            rot_application.Position
-                            - self.prev_rot_application.Position
-                        ) / dt
-
+                await self.rotator_position_updated.wait()
```

Comment:
I think this works, but I'm a bit confused about your change.

I think the following does the same thing in a slightly simpler way (and with no callback function need for rotator Application event):
```
ccw_position, ccw_velocity = await self.get_ccw_demand()
```
where get_ccw_demand is a new async method that does this:
```
rot_data = await self.rotator.tel_Application.next(flush=False)
#...do all the stuff that is in rotator_position_callback (just rename it)
# except make ccw_demand_position and ccw_demand_velocity *local* variables
# and just return them when you are done
return ccw_demand_position, ccw_demand_velocity
```